Library/Homebrew/test/cask/cask_loader_spec.rb

```diff
@@ -65,7 +65,7 @@
 
     context "when not using the API", :no_api do
       context "when a cask is migrated" do
-        let(:token) { "local-caffeine" }
+        let(:token) { "some_cask" }
```

Comment:
I have to change this because this cask exists in the core cask tap (https://github.com/Homebrew/brew/blob/cd651094878e88cb710bc24e253bff8b3595258e/Library/Homebrew/test/support/fixtures/cask/Casks/local-caffeine.rb) and these casks have special logic at https://github.com/Homebrew/brew/pull/17385/files#diff-aea1b1bc211f85168f88057b630dac95a4947b4abfa99c158ac592090fb3b23dR453

Reply:
I didn't have to duplicate it all. It was even easier. I only need cask that is not in the fixture for the new test I wrote.